src/System.Text.Json/tests/JsonStringTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Tests
+{
+    public static class JsonStringTests
+    {
+        [Fact]
+        public static void TestDefaultCtor()
+        {
+            var jsonString = new JsonString();
+            Assert.Equal("", jsonString.Value);
+        }
+
+        [InlineData("value")]
+        [InlineData("value with some spaces")]
+        [InlineData("     training spaces")]
```

Comment:
Was this meant to be "trailing spaces"? And if so, I would have expected the spaces to come after the string, not before.

nit: Add leading spaces test case too?
```C#
[InlineData("     leading spaces")]
```

Reply:
Yes, sure, I mixed the terms.